source/features/useful-forks.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {RepoForkedIcon} from '@primer/octicons-react';
+import * as pageDetect from 'github-url-detection';
+import {getRepo} from '../github-helpers';
+
+import features from '.';
```

Comment:
```suggestion
import React from 'dom-chef';
import select from 'select-dom';
import * as pageDetect from 'github-url-detection';
import {RepoForkedIcon} from '@primer/octicons-react';

import features from '.';
import {getRepo} from '../github-helpers';
```

This should also fix the lint error

Reply:
I did run the `npm run lint --fix`, but I found the message very cryptic. I don't see what is wrong with the way the imports were before and why what you did actually fixes anything.

Reply:
Okay, I think I get it: it's separating imports of external dependencies and imports from within the same source repository.